conbench/entities/summary.py

```diff
@@ -148,6 +151,18 @@ def create(data):
             bulk.append(Time(result=x, summary_id=summary.id, iteration=i + 1))
         Time.bulk_save_objects(bulk)
 
+        distribution = get_distribution(repository, sha, 1000).first()
+        if distribution:
+            values = dict(distribution)
+            Session.execute(
+                insert(Distribution.__table__)
+                .values(values)
+                .on_conflict_do_update(
```

Comment:
OOH this is nice. I punted on the sql to do this, I'm glad sql alchemy (or our infrastructure?) makes this easy!